### DISCUSSION TITLE
Support of laravel 9

### DISCUSSION THREAD

**Discussion by encodetocode** [dbfx/laravel-strapi#12] [General]
Dears,

Thanks for your hard work and good support,

I would like to ask is there going to be any new release supporting Laravel 9? 

and what version is being used now?

Thank you,

**Comment by dbfx** [comment]
Hi there

Sorry for the delay, it should be updated now with support. Please let me know any problems!

Thanks

**Comment by encodetocode** [comment]
Thank you for the fast reply, 

Sure if I had a problem would let you know, appreciate it.

Thanks.
